### PR TITLE
Fix links in footer

### LINKED ISSUE #3980: Broken links in prettier.io footer
There are some broken links in prettier.io, namely:

- User Showcase, is https://users/ instead of https://prettier.io/users
- "About" is https://prettier.io/docs//index.html
- "Usage" is https://prettier.io/docs//install.html

### PR DESCRIPTION
**Pull request by jleeothon**
Fixes #3980

I basically based this change on looking at the "Blog" link. Please let me know if there's a way to test the site, didn't get it by reading CONTRIBUTING.md.

### CODE REVIEW

**Comment by j-f1** [comment]
> Please let me know if there's a way to test the site

Now that you’ve opened a PR, you can go to https://deploy-preview-3981--prettier.netlify.com/.

**Comment by j-f1** [comment]
Can you format the code so Prettier is happy?

> Replace `⏎··············href={this.props.config.baseUrl·+·"users"}⏎············>⏎··············User·Showcase⏎············` with `·href={this.props.config.baseUrl·+·"users"}>User·Showcase`

File: website/core/Footer.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Footer extends React.Component {
           <div>
             <h5>Community</h5>
             <a
-              href={this.props.config.baseUrl + this.props.language + "/users/"}
```
Comment (ikatyang) [inline website/core/Footer.js]: I think the proper fix should be to make `this.props.language` defaults to `en` and so does `About` and `Usage`.

**Comment by jleeothon** [comment]
What about now @ikatyang

**Comment by ikatyang** [comment]
Thanks!

**Comment by jleeothon** [comment]
Thanks @ikatyang and @j-f1 =D

**Pull request merged**